rhel7/profiles/anssi_nt28_high.profile

```diff
@@ -8,4 +8,12 @@ description: 'Draft profile for ANSSI compliance at the high level. ANSSI stands
 
 extends: anssi_nt28_enhanced
 
-selections: []
+selections:
+    - var_selinux_policy_name=targeted
+    - selinux_policytype
+    - package_setroubleshoot_removed
```

Comment:
``package_setroubleshoot_removed`` should be in ``enhanced`` not ``high``

Reply:
This is actually in the Hardening level High.

Reply:
@shawndwells read through some of the document... @yuumasato is correct.